src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -439,7 +439,7 @@ namespace HVACUnitaryBypassVAV {
 
             thisCBVAV.OutAirSchPtr = ScheduleManager::GetScheduleIndex(state, Alphas(3)); // convert schedule name to pointer (index number)
             if (thisCBVAV.OutAirSchPtr != 0) {
-                if (!ScheduleManager::CheckScheduleValueMinMax(state, thisCBVAV.OutAirSchPtr, "<", 0.0, ">", 1.0)) {
+                if (!ScheduleManager::CheckScheduleValueMinMax(state, thisCBVAV.OutAirSchPtr, ">=", 0.0, "<=", 1.0)) {
```

Comment:
This is actually a bug, but it was never caught because of the cockamamie way that `CheckScheduleValue` dealt with these strings.

Reply:
We have to document this. I see if `!OutAirSchPtr` is < 0 or > 1 there is an error. This change does not seem to conform to that.
"The schedule values in {} must be 0 to 1.", cAlphaFields(3)));

What happened before and what is happening now.

Reply:
What happened before was that the `MinString` was only checked for `>`. If it was anything else, it was assumed to be `>=`. `MaxString` behavior was similar. This call intended `>=` and to get that effect it could have put anything in there except for `>`. The fact that it chose `<` is confusing/annoying/hilarious.

Reply:
Also all of these things should be changed to use the `Clusive` enumeration rather than strings. But I didn't do that in this PR, which was getting a bit out of control anyways.